patches/components-shared_highlighting-core-common-disabled_sites.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/shared_highlighting/core/common/disabled_sites.cc b/components/shared_highlighting/core/common/disabled_sites.cc
+index 65ee26acb7aae22bb3b7c6f1e64f7fc2eb3e0df8..cdc9d8c53e1873c6b6b6b1fd24d1924b9021bab0 100644
+--- a/components/shared_highlighting/core/common/disabled_sites.cc
++++ b/components/shared_highlighting/core/common/disabled_sites.cc
+@@ -17,6 +17,7 @@ namespace shared_highlighting {
+ namespace {
+ 
+ bool IsAmpGenerationEnabled() {
++  if ((true)) return false;
```

Comment:
what exactly are we disabling here? It's not clear to me what this feature even does and when we want `ShouldOfferLinkToText` to return true

Reply:
Upstream had deleted the feature flag `kSharedHighlightingAmp` and made it the default behaviour. I consulted @pes10k on this, and he asked me to keep this feature disabled.

Reply:
I pinged @pes10k about this because I still didn't quite understand why we were disabling one part of `ShouldOfferLinkToText` (and exactly what that one part is) and he said we should go ahead and disable the entire thing which I think should be doable through an override for `ShouldOfferLinkToText` instead of a patch

Reply:
Fixed in 48609580d94b2df82be68fcf7c81cfe89a050890.